builder-support/dockerfiles/Dockerfile.target.centos-8

```diff
@@ -9,12 +9,6 @@ FROM centos:8 as dist-base
 @IF [ ${BUILDER_TARGET} = centos-8-stream ]
 FROM quay.io/centos/centos:stream8 as dist-base
 @ENDIF
-@IF [ ${BUILDER_TARGET} = centos-8-amd64 ]
-FROM amd64/centos:8 as dist-base
```

Comment:
this reminds me that `centos:8` is a dead target on every arch, and the only thing still alive in -this- Dockerfile is 8-stream. But we can clean that up separately.

Reply:
Ok! I will do it now

Reply:
Done. I also changed the endpoint from where packages are retrieved for `centos:8-stream` from `mirrorlist` to `vault`.

BTW, **Centos 8 stream** reached end-of-builds a few days ago. Maybe it is just better to change it to `el-8`.

Reply:
> BTW, **Centos 8 stream** reached end-of-builds a few days ago.

oh darn, I missed that.

> Maybe it is just better to change it to `el-8`.

yes, indeed, all traces of centos 8 (normal and stream) can go then.